docs/Developers/Development_Setup.rst

```diff
@@ -11,10 +11,46 @@ Virtualenv
 
 We highly recommend installing coala in a virtualenv for development. This
 will allow you to have a contained environment in which to modify coala,
-separate from any other installation of coala that you may not want to
-break. For more information about virtualenvs, please refer to the
-`virtualenv setup <https://docs.coala.io/en/latest/Help/MAC_Hints.html#create-virtual-environments-with-pyvenv>`__ section for information on setting one
-up.
+separate from any other installation of coala that you may not want to break.
+Here we will be showing how to have a virtualenv using :code:`venv` and
+:code:`virtualenv`.
```

Comment:
If I understand corectly, `venv` and `virtualenv` is serving the same purpose? How about telling the reader to just do one of them, with `venv` as the recommended method because it's from Python standard library.